content/en/security/threats/workload_security_rules/_index.md

```diff
@@ -19,9 +19,68 @@ further_reading:
   text: "Learn more about Security notification variables"
 ---
 
-With Cloud Security Management Threats (CSM Threats) enabled, the Datadog Agent actively monitors system activity and evaluates it against a set of out-of-the-box rules to detect suspicious behavior. CSM Threats rules consist of two different components: [Agent rules](#agent-rules) and [detection rules](#detection-rules).
+This topic explains how Cloud Security Management Threats (CSM Threats) actively monitors system activity and evaluates it against a set of out-of-the-box (OOTB) rules to detect suspicious behavior.
 
-## Agent rules
+## CSM Threats rules construction
+
+CSM Threats rules consist of two different components: Agent rules and Threat detection rules.
+
+- **Agent rules:** [Agent rules][9] are evaluated on the Agent host. CSM Threats first evaluates activity within the Datadog Agent against Agent expressions to decide what activity to collect. Agent expressions use Datadog’s [Security Language (SECL)][2]. 
+  
+  For example, here is the *Agent Configuration rule* expression `cryptominer_args`: 
+
+  ```text
+  exec.args_flags in ["cpu-priority", "donate-level", ~"randomx-1gb-pages"] ||
+  exec.args in [
+      ~"*stratum+tcp*",
+      ~"*stratum+ssl*",
+      ~"*stratum1+tcp*",
+      ~"*stratum1+ssl*",
+      ~"*stratum2+tcp*",
+      ~"*stratum2+ssl*",
+      ~"*nicehash*",
+      ~"*yespower*"
+  ]
+  ```
+- **Threat Detection rules:** [Threat Detection rules][3] are evaluated on the Datadog cloud (backend). Threat Detection rules are composed of existing Agent Configuration rules and additional expression parameters.
+  
+  Here is the *Threat Detection rule* `Process arguments match cryptocurrency miner`. It uses the Agent Configuration rules, `cryptominer_args` and `windows_cryptominer_process`, identified by `@agent.rule_id`, with additional expression parameters:
+
+  ```text
+  @agent.rule_id:(cryptominer_args || windows_cryptominer_process) 
+  -@process.executable.path:"/usr/bin/grep"
+  ```
+
+### CSM Threats rules pipeline
+
+CSM Threats use the following pipeline when evaluating events:
+
+1. The Agent rules evaluate system activity on the Agent host.
+2. When activity matches an Agent rule expression, the Agent generates a detection event and passes it to the Datadog cloud (backend).
+3. The Datadog cloud evaluates the detection event to see if it matches any Threat Detection rules that use the Agent rule that sent the event.
+4. If there is a match, a signal is generated and displayed in [Signals][8].
+5. Any [Notification Rules][10] that match the severity, detection rule type, tags, and attributes of the signal are triggered.
+
+### Saving resources by design
```

Comment:
Do you think it would make more sense to have this near the top of the article? It feels like more of an overview with context on how and why CSM Threats detection rules work the way they do. The **CSM Threats rules construction** section then provides a more granular, detailed look into the individual components of the rules.

Reply:
@drichards-87 I feel like I need to introduce the components first, and then explain the pipeline. I will have an accompanying graphic, too. Waiting on it.